### ISSUE TITLE
Visualization Idea: Show cost of services per citizen

### ISSUE THREAD

**Issue by mkchandler** [codeforokc/openbudgetokc#41]
_From @mkchandler on February 24, 2015 1:38_

Example using the federal budget: http://www.nytimes.com/interactive/2014/01/19/us/budget-proposal.html

_Copied from original issue: codeforokc/okc-budget-data#5_

**Comment by brentlightsey** [comment]
@mkchandler I'll experiment with this. Might be able to reuse NYTimes code.

**Comment by mkchandler** [comment]
@brentlightsey 👍

**Comment by brentlightsey** [comment]
Working on a CodePen on this to get the output right: http://codepen.io/brentlightsey/pen/PWZNxY

**Comment by brentlightsey** [comment]
@mkchandler Matt (and others), please check out this CodePen. 
https://codepen.io/brentlightsey/pen/PWZNxY

So far just copied the styles from the NYT article. What elements would we want to include and exclude? Would definitely love input on how we want to style each part. 

Meanwhile, I'll work on the math to calculate the right relative font sizes for the numbers.

**Comment by mkchandler** [comment]
@brentlightsey We will take a look at this during our hack night tonight! Thanks! Have you joined our Slack already (http://slack.codeforokc.org)?

**Comment by brentlightsey** [comment]
@mkchandler Yes sir! I've just been reading so far, but will get in there soon. Have a commitment on Wednesdays that sometimes prevents my virtual participation. But still going to move forward on getting this into a D3 visual.

**Comment by brentlightsey** [comment]
This pen has been updated with the basic markup and javascript that I think will work OK. Let me know what you think:

http://codepen.io/brentlightsey/full/PWZNxY/

**Comment by jagthedrummer** [comment]
@brentlightsey That is looking great! Is that actually our data?

**Comment by brentlightsey** [comment]
Hey @jagthedrummer actually, this pen has the actual 2017 data in it. I'll ask for a little feedback via Slack:

http://codepen.io/brentlightsey/full/bqrpWW/
